Makefile

```diff
@@ -7,7 +7,6 @@ fix:
 	$(HERE)/vendor/bin/php-cs-fixer fix --config=$(HERE)/.php_cs
 
 lint:
-	find -L $(HERE)/src -name '*.php' -print0 | xargs -0 -n1 php -l > /dev/null
-	find -L $(HERE)/migrations -name '*.php' -print0 | xargs -0 -n1 php -l > /dev/null
+	$(HERE)/vendor/bin/php-cs-fixer fix --config=$(HERE)/.php_cs --dry-run
 
-test: fix lint
+test: lint
```

Comment:
I wonder, why don't we just have 'fix' and 'test'. Why do we have the lint option?

Reply:
`fix` actually attempts to reformat the code, whereas `lint` just fails if they can be `fix`'d and therefore not adhering to the CS - `fix` is more utility to help contributors prior to commit/PR.